hypothesis-python/src/hypothesis/internal/conjecture/engine.py

```diff
@@ -967,6 +979,16 @@ def event_to_string(self, event):
         return result
 
 
+def _is_simple_mask(mask):
+    """A "simple" mask contains only a sequence of right-aligned 1 bits, with
```

Comment:
nit: I feel like this definition could be simplified to "A simple mask is `2 ** n - 1` for some `n`, so has the effect of dropping all but the low `n` bits." or words to that effect.